pkg/kubelet/dockershim/libdocker/helpers.go

```diff
@@ -60,6 +60,13 @@ func matchImageTagOrSHA(inspected dockertypes.ImageInspect, image string) bool {
 			// hostname or not, we only check for the suffix match.
 			if strings.HasSuffix(image, tag) || strings.HasSuffix(tag, image) {
 				return true
+			} else {
+				t, _ := dockerref.ParseNormalizedNamed(tag)
```

Comment:
if this returns an error, we should continue, not ignore it

Reply:
Done